src/net.h

```diff
@@ -954,6 +958,8 @@ class CNode
         assert(false);
     }
 
+    Network ConnectedThroughNetwork() const;
```

Comment:
Can you add a doxygen comment here? Maybe provide a quick context on the difference between this and `GetNetClass()` or why it's needed.

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19998#issuecomment-703090033).